st2common/st2common/config.py

```diff
@@ -82,6 +82,14 @@ def register_opts(ignore_errors=False):
     ]
     do_register_opts(content_opts, 'content', ignore_errors)
 
+    webui_opts = [
+        cfg.StrOpt('webui_base_url', default='https://localhost',
```

Comment:
>Can we move this logic to st2chatops package / bot since it's ChatOps specific (user already needs to configure some environment variables to get bot to work)?

It was already there and there are two use cases one of them which it wasn't solving. I called it out in the description.